Python/Product/Debugger/DkmDebugger/Proxies/Structs/PyFrameObject.cs

```diff
@@ -74,24 +112,29 @@ public PyFrameObject(DkmProcess process, ulong address)
             return new PyFrameObject(frame.Process, framePtr);
         }
 
-        public PointerProxy<PyCodeObject> f_code {
-            get { return GetFieldProxy(_fields.f_code); }
+        public virtual PointerProxy<PyFrameObject> f_back {
```

Comment:
It doesn't look like you added any overrides for these properties you are making virtual. Do they need to be virtual?